docs/docs/debug/upgrade.md

```diff
@@ -0,0 +1,324 @@
+---
+id: upgrade
+title: Ory Kratos Upgrade Guide
+---
+
+This guide covers essentials to consider when upgrading Ory Kratos, please also
+refer to [UPGRADE.md](https://github.com/ory/kratos/blob/master/UPGRADE.md).
+
+Generally after upgrading just run `kratos migrate`.
+
+Please note the following breaking changes when upgrading from Ory Kratos 0.5 to
```

Comment:
I would suggest to just link to the changelog document here, otherwise we'll have to keep this doc here in snyc.